libs/test/SalesforceHybridTest/src/com/salesforce/androidsdk/phonegap/SDKInfoPluginTest.java

```diff
@@ -76,9 +79,17 @@ public void testGetSDKInfo() throws NameNotFoundException, JSONException {
 		assertTrue("Wrong bootconfig oauthScopes", sdkInfoOAuthScopes.contains("api"));
 		assertEquals("Wrong bootconfig oauthRedirectURI", bootconfig.getOauthRedirectURI(), sdkInfoBootConfig.getString("oauthRedirectURI"));
 		assertEquals("Wrong bootconfig remoteAccessConsumerKey", bootconfig.getRemoteAccessConsumerKey(), sdkInfoBootConfig.getString("remoteAccessConsumerKey"));
-		assertEquals("Wrong bootconfig androidPushNotificationClientId", bootconfig.getPushNotificationClientId(), sdkInfoBootConfig.getString("androidPushNotificationClientId"));
+        try {
+            sdkInfoBootConfig.getString("androidPushNotificationClientId");
```

Comment:
Instead of doing this, let's add a dummy value for ```androidPushNotificationClientId``` for the test project's ```bootconfig.xml```, and revert this change. The test should pass then.